lifecycle/support/infrastructure-creation-policy.md

```diff
@@ -0,0 +1,70 @@
+# Infrastructure Creation & Modification Policy
+
+## Purpose
+
+The purpose of this Infrastructure Creation & Modifcation Policy is to establish guidelines and procedures for the creation, setup, and modification of infrastructure within our organization. This policy ensures that infrastructure components are designed, provisioned, and secured in a consistent and reliable manner.
+
+## Scope
+
+This policy applies to all team members involved in the creation, setup, and modification of infrastructure within our organization, including developers, engineers, administrators, and security personnel. The policy covers the following technologies and services:
+
+- AWS for EC2, RDS, S3 Storage, CloudFront CDN, and Route 53 for domain management.
+- CircleCI and GitHub for deployment management.
+- Load.io for stress testing.
+- Detectify for scanning OWASP security issues.
+- AWS Shield for DDoS attack protection.
+- Zoonu for performing pentests.
+- Manual Pre-Launch Checklist on Confluence for new projects.
+
+## Roles and Responsibilities
+
+1. Lead Support Engineer and his team:
+   - Responsible for all infrastructure-related activities within the organization.
+   - Design and architect infrastructure components based on project requirements and best practices.
+   - Ensure scalability, availability, and security of the infrastructure.
+   - Collaborate with stakeholders to gather requirements and obtain approvals.
+   - Document the infrastructure design and configuration.
+
+2. Developers:
+   - Test their own infrastructure configurations to ensure functionality and compatibility with the application.
+   - Follow the infrastructure design guidelines provided by the Lead Support Engineer and his team.
+   - Collaborate with the infrastructure team to resolve any configuration or deployment issues.
+   - Facilitate communication with stakeholders for requested changes to infrastructure, including to provide difference in hosting cost estimates.
+
+3. Security Personnel:
+   - Collaborate with the Lead Support Engineer and his team to identify and address security concerns during the infrastructure creation process.
+   - Review the infrastructure design and provide recommendations for implementing appropriate security measures.
+   - Ensure compliance with security policies and industry standards.
+
+## Infrastructure Creation Process
+
+1. Design and Configuration:
+   - Lead Support Engineer re-reviews the hosting estimation quote provided to the client to ascertain if the infrastructure suggested still meets the requirements of the project deliverable.
+   - The Support Team design the infrastructure components, considering scalability, availability, and security.
+   - Select appropriate AWS services, such as EC2, RDS, S3 Storage, CloudFront CDN, and Route 53, based on project requirements.
+   - On Request: Document the infrastructure design, including architecture diagrams, configurations, and network layouts.

```

Comment:
I've done this because the number of times it changes during the course of the projects initial setup would mean I'd be there all day rewriting documentation